### ISSUE TITLE
How to calculate IoU with each ground truth instance?

### ISSUE THREAD

**Issue by buaa-luzhi** [zhengshou/scnn#16]
Hi, @zhengshou 
     
     Thanks for your sharing!
 
     For candidate segments from untrimmed video, how to calculate  IoU? Do you any scripts for calculation?

    Looking forward to your reply!

**Comment by zhengshou** [comment]
Thank you for your interest in our work.

Please refer to our paper section 3.2 and code on github https://github.com/zhengshou/scnn/blob/master/experiments/THUMOS14/eval/nms_temporal.m to learn about details.

**Comment by buaa-luzhi** [comment]
@zhengshou 
Thank you very much!
I have read it carefully! 
For peoposal networks, I was in trouble during the production of the training set.  I don't know how to determine the candidate segment label.  Is 1 or 0 for candidate segments label?
I just want to know how to determine training set labels for candidate segments in untrimmed videos.
Do you have any script implementation?
 
Thank you  in advance!
